draft-davidben-tls-merkle-tree-certs.md

```diff
@@ -790,57 +793,55 @@ The transparency service maintains a mirror of the CA's latest batch number, and
 
-The update server maintains the latest validity window validated to appear in all mirrors. It updates this by polling the mirrors and running the following steps:
+Authenticity:
+: The relying party only accepts validity windows that were certified by the CA
 
-1. For each mirror, fetch the latest batch number.
+Transparency:
+: The relying party only accepts validity windows covering certificates that are published in some publicly-accessible form, so that, in particular, the subject of the certificate can notice any unauthorized certificates
 
-2. Let `new_latest_batch` be the highest batch number that is bounded by the value fetched from at least half of the mirrors. Let `old_latest_batch` be the batch number of the currently stored validity window.
+How the relying party does this has no direct impact on certificate issuance ({{issuing-certificates}}) or usage ({{using}}). Different relying parties can obtain validity windows from different sources or apply different policies, while supporting the same unmodified certificates and CAs. Thus this section does not prescribe particular policies or mechanisms, but provides examples and general guidance. Relying parties MAY implement policies other than those described below, and MAY incorporate entities acting in roles not described in this document.
 
-3. If `new_latest_batch` equals `old_latest_batch`, finish this procedure without reporting an error.
+Relying parties SHOULD ensure transparency by obtaining validity windows from the CA and/or some combination of trusted mirrors. The relying party picks sources which, together, are trusted to satisfy the requirements described in {{mirrors}}. Mirrors allow the relying party to maintain transparency in the face of a misbehaving CA that may, for example, stop serving some unauthorized certificate in a batch to evade detection.
 
-4. If `new_latest_batch` is less than `old_latest_batch`, abort this procedure with an error.
+A relying party might trust a combination of mirrors, and only accept a validity window once some minimum number of mirrors have consumed it. When combining multiple mirrors, the following procedure determines the latest validity window for an update:
 
-5. If the issuance time for batch `new_latest_batch` is after the current time (see {{parameters}}), abort this procedure with an error.
+1. Fetch the latest batch number from each mirror.
 
-6. Fetch the validity window with `new_latest_batch` from each mirror that returned an equal or higher latest batch number. If any fetches fail, or if the results do not match across all mirrors, abort this procedure with an error.
+2. Compute the highest batch number that satisfies the policy. For example, if requiring windows be represented in at least two mirrors, use the second highest batch number.
```

Comment:
nit: if the two mirrors both serve the highest batch number then we don't want the second-highest
```suggestion
2. Compute the highest batch number that satisfies the policy. For example, if requiring windows be represented in at least two mirrors, use the second-to-last batch number after sorting in ascending order.
```